mobile/src/main/java/org/openhab/habdroid/core/OpenHABVoiceService.java

```diff
@@ -111,9 +112,29 @@ public void onOpenHABTracked(String baseUrl, String message) {
         stopSelf(getLastStartId());
     }
 
+    /**
+     * @param message message to show
+     * @param messageType must be Constants.MESSAGES.DIALOG
```

Comment:
Should probably throw an exception if not DIALOG?

Another thing: If that must be MESSAGES.DIALOG and because logLevel is not implemented, you probably want to add an overloaded method with the message parameter only and call this one with the only accepted values. Or, even better, remove these parameters from this method, if you do not plan to use them in the very short future (they can easily be added later, if needed).

Reply:
OpenHABVoiceService implements OpenHABTrackerReceiver, so I need a function with that parameters since they are used in MainActivity.